arcgis-ios-sdk-samples/Layers/Browse WFS layers/BrowseWFSLayers.storyboard

```diff
@@ -0,0 +1,150 @@
+                                            <fontDescription key="fontDescription" type="system" pointSize="17"/>
+                                            <nil key="textColor"/>
+                                            <nil key="highlightedColor"/>
+                                        </label>
+                                    </subviews>
+                                </tableViewCellContentView>
+                            </tableViewCell>
+                            <tableViewCell clipsSubviews="YES" contentMode="scaleToFill" preservesSuperviewLayoutMargins="YES" selectionStyle="none" indentationWidth="10" reuseIdentifier="SettingsCell" id="Zyv-ID-psG" customClass="SettingsTableViewCell" customModule="ArcGIS_Runtime_SDK_Samples" customModuleProvider="target">
+                                <rect key="frame" x="0.0" y="99.5" width="375" height="44"/>
+                                <autoresizingMask key="autoresizingMask"/>
+                                <tableViewCellContentView key="contentView" opaque="NO" clipsSubviews="YES" multipleTouchEnabled="YES" contentMode="center" preservesSuperviewLayoutMargins="YES" insetsLayoutMarginsFromSafeArea="NO" tableViewCell="Zyv-ID-psG" id="0eh-SC-W3k">
+                                    <rect key="frame" x="0.0" y="0.0" width="375" height="43.5"/>
+                                    <autoresizingMask key="autoresizingMask"/>
+                                    <subviews>
+                                        <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="Swap" textAlignment="natural" lineBreakMode="tailTruncation" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="oo0-DJ-aZi">
+                                            <rect key="frame" x="16" y="11.5" width="42" height="20.5"/>
+                                            <fontDescription key="fontDescription" type="system" pointSize="17"/>
+                                            <nil key="textColor"/>
+                                            <nil key="highlightedColor"/>
+                                        </label>
+                                        <switch opaque="NO" contentMode="scaleToFill" horizontalHuggingPriority="750" verticalHuggingPriority="750" contentHorizontalAlignment="center" contentVerticalAlignment="center" translatesAutoresizingMaskIntoConstraints="NO" id="n0O-Fe-0GS">
+                                            <rect key="frame" x="310" y="6.5" width="51" height="31"/>
+                                        </switch>
+                                    </subviews>
+                                    <constraints>
+                                        <constraint firstItem="n0O-Fe-0GS" firstAttribute="leading" relation="greaterThanOrEqual" secondItem="oo0-DJ-aZi" secondAttribute="trailing" constant="150" id="DYn-vV-8um"/>
```

Comment:
I see what's going on. The purpose of this constraint is to limit the width to which the label can grow. But with such a large constant, it is actually being used to position the switch. I'm sorry for the confusion. I obviously didn't do a good enough job explaining what this constraint was for. Because a label's width is dependent on its text, the wider the text, the wider the label. With a sufficiently large text, the label would start to overlap the switch. This constraint is meant to limit the label from doing that. It says that the distance between the two must be greater than, or equal to, the constant. The constant is meant to be a small value; I think the standard spacing is like `8`. You can use the standard value by not specifying a constant:

![Screen Shot 2019-04-01 at 3 40 41 PM](https://user-images.githubusercontent.com/2257493/55364199-926caf00-5494-11e9-852a-29ab581b618b.png)

That allows the label's width to be as small as needed to fit the text. It also allows the label to grow to fit the text up until it is `Standard` spacing away from the switch. Then the system says, "That's it! That is the maximum you're allowed to grow." 

Does that make sense?

Reply:
Ah got it 👍